custom_function_template.gs

```diff
@@ -0,0 +1,154 @@
+/**
+* A custom function template for the following functions - 
+*
+* (operation = 0) Get sum of given values 
+* @return The sum of given values 
+*
+* (operation = 1) Get mean of given values 
+* @return The mean of given values
+*
+* (operation = 2) Get variance for the given input values.
+* @return The variance for the given input values.
+*
+* (operation = 3) Get standard deviation for the given input values.
+* @return The standard deviation for the given input values.
+*
+* (operation = 4) Get all the modes for the given input values.
+* @return The modes for the given input values.
+*
+* (operation = 5) Get maximum value for the given values.
+* @return The maximum value for the given values.
+*
+* (operation = 6) Get countIf - count of input values equal to the given value 
+* @return The count of values equal to the first input value.
+*
+* (operation = 7) Get The median for the given input values.
+* @return The median for the given input values.
+*
+* @customfunction
+*/
+function customFunctionTemplate() {
```

Comment:
I feel like we should have separate custom functions for each functionality, it will be more readable both here and when used as formula?

@gargabhay what do you feel?

Reply:
True. But the idea of having a custom function template is defeated IMO. Having 5-6 custom functions might be confusing to the user. Wdyt?

Reply:
I feel the current one is too complicated for anyone to use. I was under the impression that custom function template was a way to let the user know that apps script functions could be called from the sheet.